src/Components/Form.js

```diff
@@ -34,71 +32,68 @@ function Form(props) {
     setFormData(newData);
   };
 
-  const handleBlur = (value, key) => {
-    setEmptyData((previousState) => {
-      return {
-        ...previousState,
-        [key]: value === "",
-      };
-    });
+  const handleBlur = (key) => {
+    if (key === "id") return;
+    if (formData[key] == "") {
+      document.getElementById(key).innerHTML = key + " required";
```

Comment:
why this change?

Reply:
Was told to figure out a method w/o using emptyData state

Reply:
@alexholmes99 sounds good, try doing this with `useRef` instead let me know if you need help. Check here for documentation... [here](https://reactjs.org/docs/hooks-reference.html#useref)